cmake/fw_zip.cmake

```diff
@@ -31,6 +30,13 @@ function(generate_dfu_zip)
     "type=${GENZIP_TYPE}"
     "board=${CONFIG_BOARD}"
     "soc=${CONFIG_SOC}"
+    DEPENDS ${GENZIP_TARGET}
+    )
+
+  add_custom_target(
+    ${GENZIP_TYPE}_ZIP_TARGET
```

Comment:
Why is `ZIP_TARGET` is upper case.

Secondly, executing: `ninja application_ZIP_TARGET` is a bit redundant, as `ninja [target...]` is the signature for ninja.
Which means, everything invoked with `ninja` is a build target.

So I suggest:
```suggestion
    dfu_${GENZIP_TYPE}_zip
```

the dfu is added to make the target command aligned with the output name.

Alternatively, use:
```
get_filename_component(TARGET_NAME ${GENZIP_OUTPUT} NAME)
string(REPLACE "." "_" TARGET_NAME ${TARGET_NAME})
```
to get the target name == output file, with `.` --> `_`.

Reply:
Extracted from filename, check it now.